config.h

```diff
@@ -28,6 +28,10 @@
 #define DEFAULT_TRANSMIT "1"
 #define DEFAULT_LOG_MODE "NODE" // NONE, NODE, NETWORK, ALL
 #define DEFAULT_COLLECTION_PERIOD "60" //defaults to 60 sec
+#define DEFAULT_SD_CHIP_SELECT_PIN "10"
+#define DEFAULT_RFM95_CS "10"
```

Comment:
I think we want to default to the values that would work for the integrated radio (ie. the Feather M0 Lora: https://www.adafruit.com/product/3178) For CS, RST, INT, I believe this would be 8,4,3 respectively (see https://learn.adafruit.com/adafruit-feather-m0-radio-with-lora-radio-module/pinouts and https://github.com/NUKnightLab/SensorGrid/blob/master/io.h#L15)